requirements.txt

```diff
@@ -1,2 +1,6 @@
-mock==2.0.0
-termcolor==1.1.0
+coverage
+flake8
+mock
+nose
+termcolor
+tox
```

Comment:
Yes I believe that. Doesn't mean we're future proof. So if you find the current versions are working for you, you should lock that down now.